code/controllers/configuration.dm

```diff
@@ -519,6 +521,15 @@
 					config.silicon_max_law_amount	= text2num(value)
 				if("join_with_mutant_race")
 					config.mutant_races				= 1
+				if("roundstart_races")
+					if(!cleared_default_races)
+						roundstart_species = list()
+						cleared_default_races = 1
+					var/race_name = lowertext(value)
+					for(var/speciestype in typesof(/datum/species) - /datum/species)
+						var/datum/species/S = new speciestype()

```

Comment:
I found another way around this but I'll try to keep this nugget of information better in my mind this time.